src/command.test.js

```diff
@@ -41,10 +42,41 @@ describe('Command', () => {
     })
   })
 
+  describe('with non-static schema', () => {
+    describe('and empty object schema', () => {
+      class TestCommand extends Command {
+        schema = {}
+      }
+
+      it('should fail', async () => {
+        expect(async () => await TestCommand.run()).rejects.toThrow(CommandWithNonStaticSchemaError)
+      })
+    })
+
+    describe('and null schema', () => {
+      class TestCommand extends Command {
+        schema = null
+      }
+
+      it('should fail', async () => {
+        expect(async () => await TestCommand.run()).rejects.toThrow(CommandWithNonStaticSchemaError)
+      })
+    })
+
+    describe('and non-empty schema', () => {
+      class TestCommand extends Command {
+        schema = { input1: { type: 'string', default: 'default input1 value', required: true } }
+      }
+
+      it('should fail', async () => {
+        expect(async () => await TestCommand.run()).rejects.toThrow(CommandWithNonStaticSchemaError)
+      })
+    })
+  })
+
```

Comment:
For easier legibility (because of the describe indentation)
- Empty object  `{}`
- `null`
- Non Empty `{ foo:bar}`